coordinator/internal/transitengine/crypto_test.go

```diff
@@ -0,0 +1,87 @@
+// Copyright 2024 Edgeless Systems GmbH
+// SPDX-License-Identifier: AGPL-3.0-only
+
+package transitengine
+
+import (
+	"bytes"
+	"encoding/json"
+	"io"
+	r "math/rand"
```

Comment:
Why naming the import here? Also single letter variables are usually reserved for either function receivers are variables with very limited scope